docs/jitserver_tuning.md

```diff
@@ -42,14 +42,17 @@ The client-session caches are deleted when the clients terminate, but this can h
 
  To enable this feature, specify the [`-XX:+JITServerUseAOTCache`](xxjitserveruseaotcache.md) command line option, both at the server and at the client JVM.
 
- A JITServer instance can have several AOT caches, each with its own name. This addresses the situation when client JVMs with significantly different profiles of execution use the same JITServer instance. A client JVM can indicate a specific AOT cache it wants to use by providing its name with the following command line option [`-XX:JITServerAOTCacheName=<cache_name>`](xxjitserveraotcachename.md). If the client doesn't specify a name for the AOT cache, the server uses a cache named `default`.
+ A JITServer instance can have several AOT caches, each with its own name. This addresses the situation when client JVMs with significantly different profiles of execution use the same JITServer instance. A client JVM can indicate a specific AOT cache it wants to use by providing its name with the following command-line option [`-XX:JITServerAOTCacheName=<cache_name>`](xxjitserveraotcachename.md). If the client doesn't specify a name for the AOT cache, the server uses a cache named `default`.
 
  The maximum amount of memory that all the AOT cache instances combined can use at the server is 300 MB, by default. You can change this value by using the [`-XX:JITServerAOTmx=<size>`](xxjitserveraotmx.md) option. When the cache size reaches the specified limit, new clients cannot create new AOT cache instances or add new compiled methods to the existing AOT cache instances.
 
- Current limitations:
+ Typically, each JITServer server populates its own AOT caches independently of other existing servers. To help with JITServer auto-scaling, and in particular with scaling down to 0, JITServer instances can save their AOT caches to files by setting the [`-XX:+JITServerAOTCachePersistence`](xxjitserveraotcachepersistence.md) command-line option.  JITServer instances that are started later can load the existing AOT cache files into their memory, and then continue to gradually add new AOT compiled methods. Saving an AOT cache to a file is performed periodically based on the following conditions:
+
+ - The number of extra AOT methods added to the in-memory cache is equal to the number configured with the `-Xjit:aotCachePersistenceMinDeltaMethods=<number_of_methods>` option. Default value is 200 methods, and
```

Comment:
This could still be improved in my opinion. I suggest the following:

- The number of extra AOT methods added to the in-memory cache since the last save operation is equal to or larger than the value specified by the `-Xjit:aotCachePersistenceMinDeltaMethods=<number_of_methods>` option (200 methods by default), and
- The time passed since the last save operation is equal to or larger than the value specified by the `-Xjit:aotCachePersistenceMinPeriodMs=<milliseconds>` option (10000 milliseconds by default).